### ISSUE TITLE
barcode 1D reading

### ISSUE THREAD

**Issue by lmmolinero** [gokulkrishh/qrcodescan.in#19]
It will be a good feature to read also 1D barcodes

**Comment by gokulkrishh** [comment]
@lmmolinero Thanks for suggesting, but to keep this app simple, not planning to support 1D for now. But will keep it on the bucket list for future enhancements.

**Comment by lmmolinero** [comment]
Thank you very much for your kind answer. It's a pity because I suppose it wouldn't be very difficult to detect whether it is 1D or 2D code. In any case it is an excellent application and indeed very simple, as you said. Thanks

**Comment by gokulkrishh** [comment]
@lmmolinero Yes, it was originally built as a simple QR code scanner as PWA.  But there are many apps that support 1D I guess. Try them out and see if it works for you.
